plume-models/src/medias.rs

```diff
@@ -102,19 +102,19 @@ impl Media {
     pub fn html(&self, conn: &Connection) -> Result<SafeString> {
         let url = self.url(conn)?;
         Ok(match self.category() {
-            MediaCategory::Image => SafeString::new(&format!(
+            MediaCategory::Image => SafeString::trusted(&format!(
                 r#"<img src="{}" alt="{}" title="{}">"#,
                 url, escape(&self.alt_text), escape(&self.alt_text)
             )),
-            MediaCategory::Audio => SafeString::new(&format!(
+            MediaCategory::Audio => SafeString::trusted(&format!(
```

Comment:
title can't be trusted. I don't know how to handle this one. Maybe have some way of concatenating SafeString, and do `concat(trusted("<div></div>"),new("<audio/>"))`